app/cdap/components/WorkspaceList/style.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+
+export const useStyles = makeStyles({
+  wrapper: {
+    '& *': {
+      fontFamily: "'Noto Sans', sans-serif",
+    },
```

Comment:
Is this font family getting applied? Do we need this code?

Reply:
yes we need the code...Font Family is getting applied